storage/innobase/handler/ha_innodb.cc

```diff
@@ -1000,6 +1000,22 @@ static SHOW_VAR innodb_status_variables[]= {
   (char*) &export_vars.innodb_sec_rec_cluster_reads,	  SHOW_LONG},
   {"secondary_index_triggered_cluster_reads_avoided",
   (char*) &export_vars.innodb_sec_rec_cluster_reads_avoided, SHOW_LONG},
+
+  {"scan_pages_contiguous",
+  (char*) &export_vars.innodb_fragmentation_stats.scan_pages_contiguous,
+  SHOW_LONG},
+  {"scan_pages_disjointed",
+  (char*) &export_vars.innodb_fragmentation_stats.scan_pages_disjointed,
+  SHOW_LONG},
+  {"scan_pages_total_seek_distance",
+  (char*) &export_vars.innodb_fragmentation_stats.scan_pages_total_seek_distance,
+  SHOW_LONG},
+  {"scan_data_in_pages",
+  (char*) &export_vars.innodb_fragmentation_stats.scan_data_in_pages,
+  SHOW_LONG},
+  {"scan_garbage_in_pages",
```

Comment:
Likewise re. "in_pages", also, why not replace "garbage" with "deleted_recs" or similar, to increase user intuition without having to go to docs?

Reply:
Agree

Reply:
fixed

Reply:
Still "garbage"?